src/main/kotlin/lotto/domain/LottoNumber.kt

```diff
@@ -1,6 +1,11 @@
 package lotto.domain
 
-data class LottoNumber private constructor(private val value: Int) {
+@JvmInline
+value class LottoNumber(private val value: Int) {
+    fun getValue(): Int = value
+
+    override fun toString(): String = value.toString()
```

Comment:
```suggestion
@JvmInline
value class LottoNumber(val value: Int) {
    companion object {
        internal const val MIN_RANGE_NUMBER = 1
        internal const val MAX_RANGE_NUMBER = 45
        private val LOTTO_NUMBERS = (MIN_RANGE_NUMBER..MAX_RANGE_NUMBER).associateWith(::LottoNumber)

        fun from(request: Int): LottoNumber {
            return LOTTO_NUMBERS[request] ?: throw IllegalArgumentException()
        }
    }
}
```
u can change like this

Reply:
Hi! Very nice... advice! I applied it to Lotto step2 and then to all other missions. 
I didn't need to use getter, but I did it out of habit! Thanks to Jason too!